---
source: src/validation/tests/generic_validation_tests.rs
expression: make_readable(&diagnostics)
---
SyntaxError { message: "Invalid type nature for generic argument. REAL is no Int.", range: [SourceRange { range: 429..437 }], err_no: type__invalid_nature }
SyntaxError { message: "Invalid type nature for generic argument. TIME is no Int.", range: [SourceRange { range: 465..473 }], err_no: type__invalid_nature }
SyntaxError { message: "Invalid type nature for generic argument. BYTE is no Int.", range: [SourceRange { range: 475..483 }], err_no: type__invalid_nature }
SyntaxError { message: "Invalid assignment: cannot assign 'STRING' to 'DINT'", range: [SourceRange { range: 485..492 }], err_no: var__invalid_assignment }
SyntaxError { message: "Invalid type nature for generic argument. STRING is no Int.", range: [SourceRange { range: 485..492 }], err_no: type__invalid_nature }
SyntaxError { message: "Invalid assignment: cannot assign 'CHAR' to 'DINT'", range: [SourceRange { range: 494..502 }], err_no: var__invalid_assignment }
SyntaxError { message: "Invalid type nature for generic argument. CHAR is no Int.", range: [SourceRange { range: 494..502 }], err_no: type__invalid_nature }
SyntaxError { message: "Invalid type nature for generic argument. DATE is no Int.", range: [SourceRange { range: 504..512 }], err_no: type__invalid_nature }

